src/less/components/modal/index.less

```diff
@@ -1,5 +1,5 @@
 @import (reference) 'vars';
-@import (reference) 'components/layout/index';
+//@import (reference) 'components/layout/index';
```

Comment:
Looks like the Modal component has no use for anything in the Layout component.

Go ahead and remove line 2.

Reply:
please check the changes in latest commit